src/DotNet.Testcontainers.Tests/Fixtures/Containers/Modules/AlpineFixture.cs

```diff
@@ -14,9 +14,10 @@ public AlpineFixture()
     {
     }
 
-    public Task InitializeAsync()
+    public async Task InitializeAsync()
     {
-      return Task.WhenAll(this.Container.StartAsync(), this.Container.StopAsync());
+      await this.Container.StartAsync();
```

Comment:
In the tests I got an exception in the `StopAsync` method, because the container was not started yet. And I think this makes sense as `StartAsync` and `StopyAsync` are executed here in parallel. So it is actually kind of random that it has worked with the `semaphoreslim`.

Reply:
Yep, I agree. Your suggestion is much cleaner.